### DISCUSSION TITLE
Element Connections: Expiry Date?

### DISCUSSION THREAD

**Discussion by mierla** [solspace/craft-freeform#642] [Questions & Answers] [answered]
I'm setting up a job posting form, and I want to include a date/time field for when the job listing should expire, and pass that (and the rest of the form fields) to a corresponding Job entry via Element Connections.  

Everything works great, except that I can't figure out how/if I can pass that Expiry field to the Craft entry's expiry field. 

Any suggestions? Some other clever way to capture that information?

**Comment by kjmartens** [comment, marked as answer]
EDIT: We've added support for this in Freeform 4.0.23+.

~~This currently isn't available to do, but I will see if I can have a developer add support for it next week. I'll respond here next week with an update.~~ 🙂

**Reply by mierla** [reply]
Following up on this - any word on feasibility or workarounds? Thank you!

**Reply by kjmartens** [reply]
Yes, we've added support for it and the next release of Freeform 4 will support it (likely released this week or next). 🙂

**Comment by mierla** [comment]
That's wonderful news! Thank you so much!
